cli/lib/actions/encrypt.js

```diff
@@ -40,22 +41,31 @@ module.exports = async (args, options, logger) => {
     }
 };
 
-const encrypt = async ({ secret, file, serviceAccount, namespace, kamusUrl, certFingerprint, fileEncoding }, token = null) => {
+const warnAboutNewLines = (secret, logger) => {
+    const eolIndex = secret.indexOf(os.EOL);
+    
+    if (eolIndex !== -1) {
+        logger.warn(`Secret contains newlines at index ${eolIndex}. Is this intentional?`);
```

Comment:
Maybe wait for user input?

Reply:
Deja-vu! I'll copy from #108 
If it is intentional, nothing needs to be done.
If it's not intentional, the script can't do anything about that because we shouldn't modify the secret itself in any way.
So either way, the outcome is exactly the same.
The only thing waiting for input will protect from is if the newline is unintentional AND the user doesn't read the warning.. I think that's fair enough.

Reply:
It's just phrased as a question - maybe "Secret contains newlines, continues to encrypt"?

Reply:
Alright. Addressed this.